packages/web-components/fast-foundation/src/slider-label/slider-label.ts

```diff
@@ -17,12 +18,52 @@ const defaultConfig: SliderConfiguration = {
     disabled: false,
 };
 
+const heightNumber =
+    "(var(--base-height-multiplier) + var(--density)) * var(--design-unit)";
+
 /**
  * A label element intended to be used with the {@link @microsoft/fast-foundation#Slider} component.
  *
  * @public
  */
 export class SliderLabel extends FASTElement {
+    protected horizontalStyles = css`
+        :host {
+            align-self: start;
+            grid-row: 2;
+            margin-top: -2px;
+            height: calc((${heightNumber} / 2 + var(--design-unit)) * 1px);
+            width: auto;
+        }
+        .container {
+            grid-template-rows: auto auto;
+            grid-template-columns: 0;
+        }
+    `;
+
+    protected verticalStyles = css`
+        :host {
+            justify-self: start;
+            grid-column: 2;
+            margin-left: 2px;
+            height: auto;
+            width: calc((${heightNumber} / 2 + var(--design-unit)) * 1px);
+        }
+        .container {
+            grid-template-columns: auto auto;
+            grid-template-rows: 0;
+            min-width: calc(var(--thumb-size) * 1px);
+            height: calc(var(--thumb-size) * 1px);
+        }
+        .mark {
+            transform: rotate(90deg);
+            align-self: center;
+        }
+        .label {
+            margin-left: calc((var(--design-unit) / 2) * 2px);
+            align-self: center;
+        }
+    `;
```

Comment:
These should all be added in the `fast-components` instance and not `fast-foundation`.

Reply:
moving it, thanks!